### PR TITLE
154 remove return statements that can be implicit

### PR DESCRIPTION
**Pull request by soniccyclone**

### CODE REVIEW

**Review by JamesOrson — APPROVED** [review]
Poggers

File: euler/004/problem.day [hunk 1]
```diff
@@ -3,11 +3,11 @@ import "euler/helpers.day"
 
 
 fun find_palindromic_pair (upper_bound: int) int do
-	return (_find_palindromic_pair upper_bound upper_bound 0)
+	(_find_palindromic_pair upper_bound upper_bound 0)
 end
 
 fun is_palindrome (str: cstring len: size_t) bool do
-	return (_is_palindrome str len 0)
+	 (_is_palindrome str len 0)
```
Comment (soniccyclone) [inline euler/004/problem.day:10]: Found a single extra space, whoops.

File: euler/009/problem.day [hunk 2]
```diff
@@ -17,15 +17,14 @@ end
...
 end
 
 fun find_pythagorean_triplet_product (target_sum: int64_t) int64_t do
 	let result: EuclidPairResult <- (find_euclid_pair target_sum)
 	(assert result.is_success)
 	let pair: EuclidPair <- result.pair
-	return
-		(mul pair.a
+	(mul pair.a
 		(mul pair.b
```
Comment (soniccyclone) [inline euler/009/problem.day:28]: Fix indentation

**Review by soniccyclone — COMMENTED** [review]
Need to fix whitespace

File: euler/009/problem.day (hunk 2 above)
Comment (soniccyclone) [reply euler/009/problem.day:28]: No nevermind, dont' need to fix this one

**Pull request merged**